### ISSUE TITLE
Main Menu

### ISSUE THREAD

**Issue by norriseldridge** [norriseldridge/HerosJourney#1]
Needs:
Title screen
Play Option
Background Music

Functionality:
Pressing the play button will start the game on "The Hill" or it will load your save date (if it exists) and place you there.

**Comment by norriseldridge** [comment]
Marking that last check as complete

Creating another issue for saving / loading data

Once that is complete we can come back and address this issue
